corehq/apps/change_feed/topics.py

```diff
@@ -6,6 +6,7 @@
 META = 'meta'
 
 # new models
+CASE_SQL = 'case-sql'
```

Comment:
Is the idea that we could publish changes from different datasources to the same topic and then the DocumentStore would handle the lookups etc based on the datasource?